mscore/scoreview.cpp

```diff
@@ -3167,6 +3167,29 @@ void ScoreView::startNoteEntry()
 
       is.setSegment(0);
       Note* note  = 0;
+      if (_score->selection().isNone()) {
+            // select first chordrest in first measure on first staff of current page
+            QPointF center = toLogical(QPoint(width() * 0.5, height() * 0.5));
+            Page* p = point2page(center);
```

Comment:
I am not quite sure how to find first visible measure, but if I figure that out, it might make sense to do that for any view.  I used the first measure of page rather than measure closest to center to make sure we don't change current behavior in case of brand new empty score, where note entry always starts in first measure of score even if nothing selected.